src/node_config.cc

```diff
@@ -71,13 +71,22 @@ static void InitConfig(Local<Object> target,
 
   if (config_experimental_modules) {
     READONLY_BOOLEAN_PROPERTY("experimentalModules");
-    if (!config_userland_loader.empty()) {
+    if (!config_userland_loaders.empty()) {
+      auto loaders = v8::Array::New(isolate, config_userland_loaders.size());
+      for (size_t i = 0; i < config_userland_loaders.size(); i++) {
+        auto item = config_userland_loaders[i];
+        loaders->Set(
```

Comment:
Can you use the overload that takes a `Local<Context>` and call .FromJust() on the return value?

Reply:
fixed